### PR TITLE
Fix Responsive Layout & Styling V2

### PR DESCRIPTION
**Pull request by math-marcellino**
## 1. Responsive Navigation
- Implement new navigation bar + responsive behaviour

## 2. Implement Jumbotrons in Layout
Homepage V2 now contains Jumbotron 1-3 + footer
NOTE:
- This PR also contains the `Third Jumbotron`, which actually is still WIP. Complete version of `Third Jumbotron` will be provided in different PR (RIS-108)

TODO:
- Implement gradients in `Third Jumbotron`

## 3. Configure Typography and Font
Typography is implemented as classes in `globals.css`

## 4. Preview from Safari Browser on Iphone
WhatsApp Image 2022-06-07 at 7 32 50 PM

### CODE REVIEW

File: modules/homePage/component/v2/SecondJumbotron/SecondJumbotron.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { SecondJumbotronSection } from "./Section";
 
 function SecondJumbotron() {
     return (
-        <div className="mx-auto max-w-7xl pt-32 ">
+        <div className="mx-auto max-w-[343px] pt-32 sm:max-w-[552px] lg:max-w-[936px] xl:max-w-[1128px]">
```
Comment (asaadam) [inline modules/homePage/component/v2/SecondJumbotron/SecondJumbotron.tsx:7]: from where did you get this number? 
just courius

**Review by asaadam — APPROVED** [review]
LGTM!

File: modules/homePage/component/v2/SecondJumbotron/SecondJumbotron.tsx (hunk 1 above)
Comment (math-marcellino) [reply modules/homePage/component/v2/SecondJumbotron/SecondJumbotron.tsx:7]: max width is defined in figma sir hehe. For example this one:
image

**Review by asaadam — CHANGES_REQUESTED** [review]
image

ups this section looks like not responsive

**Comment by math-marcellino** [comment]
> image
> 
> ups this section looks like not responsive

@asaadam  Hmm if we look at the design in figma, it was designed this way for the mobile version:
image

cc @ayikfour

**Comment by ayikfour** [comment]
Guys sorry just read this up. I forgot to write the specs.

I've been thinking about using HTML Marquee to show the metrics endlessly on mobile

here's the prototype https://www.figma.com/proto/nfbswhv2r8e2g7e4OPAyBw/Website-2.0?node-id=447%3A5914&viewport=-952%2C-36%2C0.53&scaling=min-zoom&starting-point-node-id=997%3A17986&show-proto-sidebar=1

I know this is deprecated on some, but can we handle it with JS? or even still use it with the default HTML Marquee tag? https://www.w3schools.in/html/marquee-tag

**Comment by math-marcellino** [comment]
> Guys sorry just read this up. I forgot to write the specs.
> 
> I've been thinking about using HTML Marquee to show the metrics endlessly on mobile
> 
> here's the prototype https://www.figma.com/proto/nfbswhv2r8e2g7e4OPAyBw/Website-2.0?node-id=447%3A5914&viewport=-952%2C-36%2C0.53&scaling=min-zoom&starting-point-node-id=997%3A17986&show-proto-sidebar=1
> 
> I know this is deprecated on some, but can we handle it with JS? or even still use it with the default HTML Marquee tag? https://www.w3schools.in/html/marquee-tag

I tried to use `marquee` HTML tag, but seems like it's not supported by JSX.
image

But I also found a `marquee` library for React: https://www.react-fast-marquee.com/
@asaadam wdyt? Can we use this?

cc @ayikfour

**Comment by ayikfour** [comment]
let me know if its possible or not guys, if it's impacting bundle size I could provide the static one

**Review by asaadam — APPROVED** [review]

**Pull request merged**